rosetta-source/src/main/rosetta/product-asset-type.rosetta

```diff
@@ -590,6 +590,7 @@ type BondReference: <"Reference to a bond underlier to represent an asset swap o
 	bond ProductIdentifier (1..1) <"Reference to a bond underlier.">
 	conditionPrecedentBond boolean (1..1) <"To indicate whether the Condition Precedent Bond is applicable. The swap contract is only valid if the bond is issued and if there is any dispute over the terms of fixed stream then the bond terms would be used.">
 	discrepancyClause boolean (0..1) <"To indicate whether the Discrepancy Clause is applicable.">
+	couponRate FixedRateSpecification (0..1) <"Specifies the coupon rate (expressed in percentage) of a fixed income security or convertible bond.">
```

Comment:
BondReference - Coupon Rate

**Background:** 
This release adds the field"couponRate" to the bondReference object under InterestRatePayout. 
The path for this field would be the following:

- InterestRatePayout > bondReference > couponRate. couponRate should be of type "FixedRateSpecification" with a cardinality of (0..1)

**Review Directions**:
- In the CDM Portal, select the Textual Browser and inspect each of the changes identified above.